docker/zap-api-scan.py

```diff
@@ -295,7 +296,7 @@ def main(argv):
     if port == 0:
         port = get_free_port()
 
-    logging.debug('Using port: ' + str(port))
+    logging.debug('Using port: %s', port)
```

Comment:
the logger takes care calling __str__ or __repr__ depending on `%s` or `%r`

Reply:
Any reason why not using f-string syntax ?

Reply:
I used to create strings only if logs are actually printed. Probably in this case it is ok using f-string.